tests/sentinel/tests/13-info-command.tcl

```diff
@@ -0,0 +1,107 @@
+# Check the basic monitoring and failover capabilities.
+source "../tests/includes/init-tests.tcl"
+
+
+test "info command with only one argument" {
+    set info [S 0 info]
+    assert { [string match "*redis_version*" $info] }
+    assert { [string match "*maxclients*" $info] }
+    assert { [string match "*used_cpu_user*" $info] }
+    assert { [string match "*sentinel_tilt*" $info] }
+    assert { ![string match "*used_memory*" $info] }
+    assert { ![string match "*rdb_last_bgsave*" $info] }
+    assert { ![string match "*master_repl_offset*" $info] }
+    assert { ![string match "*cluster_enabled*" $info] }
+}
+
+test "info all command" {
+    set info [S 0 info all]
+    assert { [string match "*redis_version*" $info] }
+    assert { [string match "*maxclients*" $info] }
+    assert { [string match "*used_cpu_user*" $info] }
+    assert { [string match "*sentinel_tilt*" $info] }
+    assert { ![string match "*used_memory*" $info] }
+    assert { ![string match "*rdb_last_bgsave*" $info] }
+    assert { ![string match "*master_repl_offset*" $info] }
+    assert { ![string match "*cluster_enabled*" $info] }
+}
+
+test "info default command" {
+    set info [S 0 info default]
+    assert { [string match "*redis_version*" $info] }
+    assert { [string match "*maxclients*" $info] }
+    assert { [string match "*used_cpu_user*" $info] }
+    assert { [string match "*sentinel_tilt*" $info] }
+    assert { ![string match "*used_memory*" $info] }
+    assert { ![string match "*rdb_last_bgsave*" $info] }
+    assert { ![string match "*master_repl_offset*" $info] }
+    assert { ![string match "*cluster_enabled*" $info] }
+}
+
+test "info everything command" {
+    set info [S 0 info everything]
+    assert { [string match "*redis_version*" $info] }
+    assert { [string match "*maxclients*" $info] }
+    assert { [string match "*used_cpu_user*" $info] }
+    assert { [string match "*sentinel_tilt*" $info] }
+    assert { ![string match "*used_memory*" $info] }
+    assert { ![string match "*rdb_last_bgsave*" $info] }
+    assert { ![string match "*master_repl_offset*" $info] }
+    assert { ![string match "*cluster_enabled*" $info] }
+}
```

Comment:
these 4 segments are nearly identical, let's run them in a for loop, each time using a different input, with the same validations.